pages/5_Compiling.py

```diff
@@ -1,15 +1,33 @@
 import streamlit as st
 
+from tflm_hello_world.training import model_training
+
 st.set_page_config(
     page_title = 'Compiling',
     page_icon = '✅',
     layout = 'wide'
 )
 
+if "compile" not in st.session_state:
+    st.session_state["compile"]="not done"
 
 st.title('Compiling')
 st.header('Compiling Hello World TinyML model')
 st.subheader('with TFLm tool')
 st.markdown('- Choose compilation options')
-st.markdown('- Compile')
+st.markdown('- Compile into a deployable Docker image')
 st.markdown('- Show the result')
+
+
+def change_compile_state():
+    st.session_state["compile"]="done"
+
+st.button("Compile!", on_click=change_compile_state)
+info = st.text("Status: waiting...")
+
+if st.session_state["compile"] == "done":
+    trainer = model_training()
+    trainer.convert_to_c_array()
```

Comment:
c-array may depends on OS installation. Some device or RTOS can take tflite directly. c_array should be in `installing`.

Reply:
Ok, So should the compilation section then include the conversion from regular keras model into a tflite model? and then .cc array conversion is an optional part in "installing" that depends on the deployment target? 😄

Reply:
Yes, that's right!